### ISSUE TITLE
Explain never shown

### ISSUE THREAD

**Issue by yazun** [sasha-alias/sqltabs#131]
Quite often explain returns no results, i.e. we just see "# Executing... 1.5s" and no output/blank output pane. 
In general v 1.0.0 seems to have more issues then older ones (0.18) - it has happened in the past (0.18) , but now much more often the view get saturated and only exit helps..

**Comment by sasha-alias** [comment]
fix of `explain` is on the way, thanks for reporting

**Comment by do-more** [comment]
how much time you'll need to release this fix? 
It's seven months without one of the most important features.

**Comment by sasha-alias** [comment]
yeah, I agree that sucks. Unfortunately the release process is quite complicated here considering the number of supported platforms and database engines. I already failed few times with releasing, so waiting for next window of enough free time.
This particular issue is fixed in the master branch. So if you are comfortable with building it from sources I'd recommend this way as the quickest possible way of getting recent fixes.
